config/vufind/markdown.ini

```diff
@@ -12,6 +12,30 @@
 ;renderer[inner_separator] = "\n"
 ; String to use for rendering soft breaks
 ;renderer[soft_break] = "\n"
+; The maximum nesting level for blocks (default: infinite). Setting this to a
+; positive integer can help protect against long parse times and/or segfaults if
+; blocks are too deeply-nested.
+;max_nesting_level = 10
+
+; Which extension you want to activate. List of extension names separated by comma.
+; Available extensions as of league/commonmark version 2.2:
+;   Attributes, Autolink, DefaultAttributes, DescriptionList, DisallowedRawHtml,
+;   ExternalLink, Footnote, FrontMatter, HeadingPermalink, Mention, SmartPunct,
+;   Strikethrough, Table, TableOfContents, TaskList.
+; More about extensions: https://commonmark.thephpleague.com/2.2/extensions/overview/
+; Some of them could have a configuration, see sections below.
+; Note: FrontMatter does enable to inject the title other then main heading. Just add
+; something like this at the very beginning of your markdown file:
+;---
+;title: "Some specific title"
+;---
+; See more about FrontMatter extension here:
+; https://commonmark.thephpleague.com/2.2/extensions/front-matter/
+
```

Comment:
I would remove this blank line to clarify the relationship between the comment above and the setting below.

Reply:
I've removed it.